src/data/hashMap/PLHashMap.spec.ts

```diff
@@ -20,7 +20,10 @@ describe('stdlib/data/PLHashMap', () => {
     it('should return with the JS representation', () => {
       const js = plHashMap(plString('a'), plNumber(1), plString('b'), plNumber(2)).toJS()
       expect(Array.from(js.keys())).toEqual(['a', 'b'])
-      expect(Array.from(js.values())).toEqual([1, 2])
+      expect(Array.from(js.values())).toEqual([
```

Comment:
this change is still not valid

Reply:
I tried to run the code in the link but it didn't work. This is my history:

![kép](https://user-images.githubusercontent.com/38993315/111056806-07520d00-8450-11eb-9eb9-07c5aa09e23d.png)

I will try to revert the changes base on [this](https://stackoverflow.com/questions/1463340/how-to-revert-multiple-git-commits).